mitielib/src/text_categorizer.cpp

```diff
@@ -71,13 +71,23 @@ namespace mitie
         string& text_tag,
         double& text_score
     ) const {
+        predict(sentence, text_tag, text_score, fe);
+    }
+
+    void text_categorizer::
+    predict (
+        const std::vector<std::string>& sentence,
+        string& text_tag,
+        double& text_score,
+        const total_word_feature_extractor& fe_
+    ) const {
```

Comment:
Try to use the same formatting as the rest of the code, i.e. put { on it's own line.